src/emulator/dataconnectEmulator.ts

```diff
@@ -102,14 +104,21 @@ export class DataConnectEmulator implements EmulatorInstance {
           `FIREBASE_DATACONNECT_POSTGRESQL_STRING is set to ${clc.bold(connStr)} - using that instead of starting a new database`,
         );
       } else if (pgHost && pgPort) {
-        const pgServer = new PostgresServer(dbId, "postgres");
-        const server = await pgServer.createPGServer(pgHost, pgPort);
+        const dataDirectory = this.args.config.get("emulators.dataconnect.dataDirectory");
```

Comment:
What if this is not set?

Reply:
It will be undefined, and pglite will fall back to using an in-memory DB instead of a file backed one.